packages/presentation/lib/main_bloc.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:bloc/bloc.dart';
+import 'package:domain/usecase/check_number_usecase.dart';
+import 'package:domain/usecase/generate_number_usecase.dart';
+import 'package:flutter/material.dart';
+import 'main_state.dart';
+import 'main_event.dart';
+
+class MainBloc extends Bloc<MainEvent, MainState> {
+  final checkNumberUseCase = CheckNumberUseCase();
+  final generateNumberUseCase = GenerateNumberUsecase();
+  bool? isGuessed;
+  final myTextController = TextEditingController();
+  int number = 1;
+  int randomNumber = 0;
+  int counter = 0;
+  bool isButtonDisabled = true;
+  static const maxAttempts = 3;
```

Comment:
cleanup

place static first
then final
then bool
then int

it will improve readability